korman/exporter/material.py

```diff
@@ -194,6 +205,78 @@ def export_waveset_material(self, bo, bm):
         # Wasn't that easy?
         return hsgmat.key
 
+
+    def export_bumpmap_slot(self, bo, bm, hsgmat, slot, idx):
```

Comment:
I'm extremely concerned with the code duplication here. I think it would be better to change `export_texture_slot` to do what you need.